pkgs/tools/video/untrunc/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, gcc, libav_12, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "untrunc-${version}";
+  version = "2018.01.13";
+
+  src = fetchFromGitHub {
+    owner = "ponchio";
+    repo = "untrunc";
+    rev = "3a2e6d0718faf06589f7b9d95c8f966348e537f7";
+    sha256 = "03ka4lr69k7mikfpcpd95smzdj62v851ididnjyps5a0j06f8087";
+  };
+
+  buildInputs = [ gcc libav_12 ];
+
+  # Untrunc uses the internal libav headers 'h264dec.h' and 'config.h'.
+  # The latter must be created through 'configure'.
+  libavConfiguredSrc = libav_12.overrideAttrs (oldAttrs: {
+    name = "libav-configured-src";
+    outputs = [ "out" ];
+    phases = [ "unpackPhase" "configurePhase" "copyConfiguredSrc" ];
```

Comment:
`patchPhase` should be added here as well.

Reply:
Should I rename `copyConfiguredSrc` to just `installPhase`?

Reply:
Yes, that would be less confusing to people.

Reply:
Done. I've fixed all remaining quirks.